module/signature/checksum.go

```diff
@@ -0,0 +1,74 @@
+package signature
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/onflow/flow-go/model/flow"
+)
+
+// CheckSumLen is fixed to be 16 bytes, which is also md5.Size
+const CheckSumLen = 32
+
+// CheckSumFromIdentities returns checksum for the given identities
+func CheckSumFromIdentities(identities []flow.Identifier) [CheckSumLen]byte {
+	return flow.MakeID(EncodeIdentities(identities))
```

Comment:
I suggest to use a CRC of 32 bits as a checksum since the purpose is only to detect honest mistakes (bugs).  You can use this function to return the checksum : https://pkg.go.dev/hash/crc32@go1.18.1#ChecksumIEEE.
It returns a `uint32` but you can convert it into `[]byte` if needed.